wallet/src/de/schildbach/wallet/ui/LockScreenActivity.kt

```diff
@@ -529,10 +529,13 @@ open class LockScreenActivity : SecureActivity() {
         fragmentManager.fragments
             .takeIf { it.isNotEmpty() }
             ?.forEach { fragment ->
-                if (fragment is DialogFragment) {
-                    fragment.dismissAllowingStateLoss()
-                } else if (fragment is NavHostFragment) {
-                    dismissDialogFragments(fragment.childFragmentManager)
+                // check to see if the activity is valid and fragment is added to its activity
+                if (fragment.activity != null && fragment.isAdded) {
+                    if (fragment is DialogFragment) {
+                        fragment.dismissAllowingStateLoss()
+                    } else if (fragment is NavHostFragment) {
+                        dismissDialogFragments(fragment.childFragmentManager)
+                    }
```

Comment:
The goal is to fix this crash.  The solution here is to check `fragment.isAdded`

```
Fatal Exception: java.lang.IllegalStateException
Fragment ReceiveDetailsDialog{5fb3ac3} (f9324875-6305-4670-8f78-6a1eb97ebb46) not associated with a fragment manager.
  androidx.fragment.app.Fragment.getParentFragmentManager (Fragment.java:1040)
  com.google.android.material.bottomsheet.BottomSheetDialogFragment.dismissAllowingStateLoss (BottomSheetDialogFragment.java:54)
  de.schildbach.wallet.ui.LockScreenActivity.dismissDialogFragments (LockScreenActivity.kt:518)
  de.schildbach.wallet.ui.LockScreenActivity.onLockScreenActivated (LockScreenActivity.kt:510)
  de.schildbach.wallet.ui.LockScreenActivity.onLogoutListener$lambda-1 (LockScreenActivity.kt:141)
  de.schildbach.wallet.ui.LockScreenActivity.lambda$eS7hrU2lCra4Z7LuRF5JiXGFef0 (LockScreenActivity.kt)
  de.schildbach.wallet.ui.-$$Lambda$LockScreenActivity$eS7hrU2lCra4Z7LuRF5JiXGFef0.onLogout (lambda:2)
  de.schildbach.wallet.AutoLogout$2.run (AutoLogout.java:92) 
```